editor/package.json

```diff
@@ -132,6 +132,7 @@
     "fontfaceobserver": "2.1.0",
     "friendly-words": "1.1.10",
     "graphlib": "2.1.1",
+    "he": "^1.2.0",
```

Comment:
Please can you remove the `^` from the dependencies that have been added here. It doesn't look like this library is likely to be updated in the near future anyway, but we've had quite a few problems when building with dependencies that weren't pinned to a specific version